open-window.csl

```diff
@@ -260,36 +293,44 @@
                   </date>
                 </else>
               </choose>
-              <text variable="page"/>
+              <text variable="page" prefix=":"/>
             </group>
           </else-if>
-          <else-if type="article-newspaper" match="any">
+          <else-if type="article-newspaper article-magazine" match="any">
             <text macro="title"/>
-            <group delimiter=" ">
-              <text variable="container-title" font-style="italic"/>
-              <text variable="publisher-place" prefix="(" suffix=")"/>
-            </group>
-            <group delimiter=": ">
-              <date variable="issued">
-                <date-part name="day" suffix=" "/>
-                <date-part name="month" form="long"/>
-              </date>
-              <text variable="page"/>
+            <group>
+              <text variable="container-title" font-style="italic" suffix=" "/>
+              <choose>
+                <if match="any" variable="issued">
+                  <date variable="issued">
+                    <date-part name="day" form="numeric-leading-zeros"/>
+                    <date-part name="month" prefix=" "/>
+                  </date>
+                </if>
+                <else>
+                  <text term="no date" form="short"/>
+                </else>
+              </choose>
+              <text variable="page" prefix=":"/>
             </group>
           </else-if>
           <else-if type="thesis" match="any">
             <text macro="title"/>
             <text variable="genre"/>
-            <text variable="publisher"/>
+            <text macro="publisher"/>
           </else-if>
-          <else-if type="chapter paper-conference entry-encyclopedia entry-dictionary" match="any">
+          <else-if type="chapter entry-encyclopedia entry-dictionary" match="any">
             <text macro="title"/>
             <group delimiter=" ">
               <text term="in" text-case="capitalize-first"/>
```

Comment:
Book chapters look wrong -- should be lowercase in, lowercase edited by and commas between elements, right?

Reply:
So, funny story. They were wrong, but last week we revised our style guide, and we've actually changed it to be consistent with how it is formatted here: _Title_. Edited by J Smith. It's more consistent with translated works and interviews now. So this format is now correct.